subt/bundle_rgbd.py

```diff
@@ -25,7 +25,10 @@ def __init__(self, config, bus):
                         *[math.radians(x) for x in camera_config['ypr']]))
         else:
             self.camera_pose = None
-        self.robot_pose = None
+        if config.get("ignore_robot_pose"):
```

Comment:
why the pose has to be ignored if it is coming from tracking camera?

Reply:
This is for cases, where the robot pose is not available e.g. testing. Also, i think that can come some applications (and they coming, e.g. plant scanning) without the robot pose in the future.

Reply:
well the bundler is combining `robot_pose` with other synchronized sources. So the proper solution (if you do not have tracking camera or other `pose3d` source) is add fake in your configuration, not into the bundler?? Also the position is not valid, so alternative would be to publish `None` for `robot_pose` . If your JSON config is correct then it should work without this patch.

Reply:
Ok. I will try to keep the "ignore_robot_pose" and then the resulting behaviour will be rgbd published  with none for robot pose. Ok?

> If your JSON config is correct then it should work without this patch.

If the `robot_pose` is unknown the bundler does not publish rgbd.

Reply:
@jisa do you want to comment on this? Bundler is primarily for delayed artifact detection and this somehow seems strange to me ... I do not know.

Reply:
The last commits contains a parameter `unknown_robot_pose`. Then the bundler publish rgbd with None for robot_pose.